rotkehlchen/chain/ethereum/modules/curve/decoder.py

```diff
@@ -115,11 +138,12 @@ def _decode_curve_remove_events(
         # 3. Withdrawal 2
         # etc.
         if return_event is None or len(withdrawal_events) == 0:
-            log.debug(
-                f'Expected to see a receive pool token event and deposit events for a curve pool, '
-                f'but have not found them. Tx_hash: {transaction.tx_hash.hex()} '
-                f'User address: {user_address}',
-            )
+            if user_address not in CURVE_DEPOSIT_CONTRACTS:
```

Comment:
It is to prevent a false positive in the logs since the behaviour of not seeing receive pool event & deposit events for a deposit contract is expected.